default/customizations/custom_sitreps.txt

```diff
@@ -190,9 +190,41 @@ EffectsGroup
 //             ]
 //             empire = Source.Owner
 
+//#####     RANDOM BEGINNER HINTS     #####//
 
-
-
+EffectsGroup
+    scope = Source
+    activation = Turn low = 0 high = 200
```

Comment:
Just notice this... should it really be shown every turn, especially if there are only 19 hits?